website/app/components/doc/cards/card.hbs

```diff
@@ -6,6 +6,12 @@
 <li class={{this.classNames}} ...attributes>
   {{#if @route}}
     <LinkTo class="doc-cards-card__link" @route={{@route}} @model={{@model}}>
+      {{#if @status.deprecated}}
+        <Doc::Badge class="doc-cards-card__badge" @type="warning-inverted" @size="medium">Deprecated</Doc::Badge>
+      {{/if}}
+      {{#if @status.updated}}
```

Comment:
non-blocker: considering we can't have two statuses, this might as well be an elseif
```suggestion
      {{else if @status.updated}}
```

Reply:
For some reasons, I thought in handlebars there was no `else if` 🤦‍♂️